### DISCUSSION TITLE
Issue with photo_demo example

### DISCUSSION THREAD

**Discussion by skvenkat18** [Seeed-Solution/SenseCAP_Indicator_ESP32#51] [Q&A]
Hello all,

I have setup the ESP IDF dev environment in VS code as per the instructions provided in the Wiki. I could run the squareline demo, lvgl demo and indicator basis demo examples with out any trouble. But when I build and flash the photo demo example only white screen appears eventhough the ESP_LOGI shows that it could access the image:

_I (801) bsp boards: detecting board SenseCAP_Indicator
I (807) bsp boards: Detected board: [SenseCAP_Indicator]
I (818) gpio: GPIO[38]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
I (842) gpio: GPIO[45]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (857) gpio: GPIO[41]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0
I (857) gpio: GPIO[48]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0
I (1269) lvgl_port: Add TP input device to LVGL
I (1282) lv_fs: Initializing SPIFFS as demo assets storage.
I (1393) lv_fs: Partition size: total: 4811921, used: 783371
--- list ---
mountain.png
scenery.png
Sunset.png
I (1445) :  Display file:mountain.png...

Please let me know If I need to change any default settings to make it work. Thanks !!

**Comment by anil-rai-demo** [comment]
/*PNG decoder library*/
#define LV_USE_PNG 1

try this in lv_conf.h file. (change the menucongif to use custom lv_conf.h)
